src/lib/replication/repl_dev/raft_repl_dev.cpp

```diff
@@ -165,6 +171,7 @@ AsyncReplResult<> RaftReplDev::replace_member(replica_id_t member_out_uuid, repl
             auto err = m_state_machine->propose_to_raft(std::move(rreq));
             if (err != ReplServiceError::OK) {
                 LOGERROR("Replace member propose to raft failed {}", err);
+                reset_quorum_size(0);
```

Comment:
we also need reset_quorum_size back to 0 in 152 and 132 before we return an error

Reply:
Ack